pyanaconda/modules/common/structures/storage.py

```diff
@@ -453,3 +453,61 @@ def get_root_device(self):
         :return: a device name or None
         """
         return self.mount_points.get("/")
+
+
+class PlatformMountPointData(DBusData):
+    """Platform required mount points data"""
+
+    def __init__(self):
+        self._value = ""
+        self._filesystem = ""
+        self._encrypted = False
+        self._lv = False
+
+    @property
+    def value(self) -> Str:
+        """Mount point value
+
+        :return: a string with mount point
+        """
+        return self._value
+
+    @value.setter
+    def value(self, value: Str):
+        self._value = value
+
+    @property
+    def fstype(self) -> Str:
```

Comment:
...actually, the names should be more descriptive, so something like `required_file_system_type` would make more sense.